.github/workflows/nightlies.yml

```diff
@@ -38,8 +38,9 @@ jobs:
       - uses: goreleaser/goreleaser-action@v6
         with:
           distribution: goreleaser-pro
-          version: v1.26.2-pro
-          args: release --clean --nightly --config ./.github/goreleaser-nightly.yaml
+          version: ~> v2
```

Comment:
do we need to keep the -pro suffix to still use the pro version?

Reply:
I think this was dropped starting from goreleaser-action@v6 for v2, where it is enough distribution: goreleaser-pro.

This is an example output from my own pipeline:

```
Run goreleaser/goreleaser-action@v6
Downloading https://github.com/goreleaser/goreleaser-pro/releases/download/v2.1.0-pro/goreleaser-pro_Linux_x86_64.tar.gz
Extracting GoReleaser
/usr/bin/tar xz --warning=no-unknown-keyword --overwrite -C /home/runner/work/_temp/abf81a88-6780-4334-88a2-a4360729df83 -f /home/runner/work/_temp/59baab70-2670-45bc-af57-2e89b4b172d5
GoReleaser ~> v2 installed successfully
/opt/hostedtoolcache/goreleaser-action/2.1.0-pro/x64/goreleaser release --clean --skip publish --skip validate --config ./.github/goreleaser.yaml
```